packages/core/parcel/src/cli.js

```diff
@@ -179,7 +179,12 @@ async function run(entries: Array<string>, command: any) {
     return;
   }
   let Parcel = require('@parcel/core').default;
-  let options = await normalizeOptions(command);
+  let options;
+  try {
+    options = await normalizeOptions(command);
+  } catch (e) {
+    return process.exit(-1);
```

Comment:
Shouldn't this be `process.exit(1)` ?

Reply:
I changed it cause otherwise we would exit with status code 0 - if there's a better way to handle that, please let me know! 😄

Reply:
Yeah, I believe more than just that line can fail, opened a seperate PR for it, https://github.com/parcel-bundler/parcel/pull/5517